docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -93,6 +93,22 @@ Using a filter you retain all information::
 General aggregate functions
 ---------------------------
 
+.. function:: any_value(x) -> [same as input]
+
+    Returns an arbitrary non-null value from all the rows of column ``x``. This
```

Comment:
@martint .. from what I can see on other mentions of function .. x can actually be an arbitrary expression. Is this the case in Trino?

Reply:
Yes, it can be any expression, just like with any other function that takes an argument. BTW, any_value is the same as the “arbitrary” function.

Reply:
Is seems to be .. for exmaple this query works

SELECT sum(o.totalprice) as spend,
	c.custkey,
any_value(substring(c.name, 9)) as customernumber
FROM tpch.tiny.orders o
JOIN tpch.tiny.customer c
ON o.custkey  = c.custkey
GROUP BY c.custkey;
ORDER BY spend;

Reply:
Not sure if this is clear from the docs on this page overall. I added in this context for now. Also saw that arbitrary is the same .. but I think we should add more useful info.